### PR TITLE
feat: add json credentials file for Google Sheets

### PR DESCRIPTION
**Pull request by redouane59**
Removing the yaml file from twitter client and use a local json file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,25 @@
 # twitter-personal-analyzer-bot
 Analyze your interactions.
 
+## Set up
+In `src\main\resources\google-credentials.json`, create the following file :
```
Comment (hypr2771) [inline README.md:5]: This does not make sense, your users are most likely not going to build your project, hence resources cannot be available and read from this folder.
Why not read from the root folder, provide an argument to the CLI command or create a CLI method to init a file from where you'll read data?

File: README.md [hunk 2]
```diff
@@ -1,6 +1,25 @@
...
+
+```$xslt
```
Comment (hypr2771) [inline README.md:7]: Isn't the type `json`?

File: src/main/java/com/socialmediaraiser/twitterbot/GoogleAuthorizeUtil.java [hunk 3]
```diff
@@ -27,20 +28,15 @@ private GoogleAuthorizeUtil() {
...
-    if (yamlFile == null) {
-      LOGGER.severe(() -> "yaml file not found at /RedTheOne.yaml");
-      return null;
+    // @todo use a local json instead
+    URL googleCredentialsFile = GoogleAuthorizeUtil.class.getClassLoader().getResource("google-credentials.json");
+    if (googleCredentialsFile == null) {
+      LOGGER.severe(() -> "file not found");
     }
-    ObjectMapper mapper = new ObjectMapper(new YAMLFactory());
+    ObjectMapper mapper = new ObjectMapper();
```
Comment (hypr2771) [inline src/main/java/com/socialmediaraiser/twitterbot/GoogleAuthorizeUtil.java:36]: Don't instantiate `ObjectMapper` each time.
Use the singleton and call the `readerFor` and `writerFor` methods against it.

File: src/main/java/com/socialmediaraiser/twitterbot/PersonalAnalyzerLauncher.java [hunk 4]
```diff
@@ -22,12 +22,14 @@ public static void main(String[] args){
...
+        };
+        String[] whiteList = {"mohamedmoh59","NC_69_","DadouDads","fayce_b", "InesInesz", "caroletonneau", "KayiTurk_fr", "Asiorak", "Sarah_Bcf", "vivi83032363"
```
Comment (hypr2771) [inline src/main/java/com/socialmediaraiser/twitterbot/PersonalAnalyzerLauncher.java:28]: You're going to have some issues here... 🙀

File: README.md (hunk 2 above)
Comment (redouane59) [reply README.md:7]: well seen !

File: src/main/java/com/socialmediaraiser/twitterbot/PersonalAnalyzerLauncher.java (hunk 4 above)
Comment (redouane59) [reply src/main/java/com/socialmediaraiser/twitterbot/PersonalAnalyzerLauncher.java:28]: Hahaha sure, I should probably keep a json user list locally

File: README.md (hunk 1 above)
Comment (redouane59) [reply README.md:5]: I just had in mind the same way that the TwitterClient is using the `twitter-credentials.json` data.
But i totally agree, the best solution should be to provide some argument, but as far as I need like 10 parameters to access the Google Sheets API, does it make sense to have so many args ?

File: src/main/java/com/socialmediaraiser/twitterbot/GoogleAuthorizeUtil.java (hunk 3 above)
Comment (redouane59) [reply src/main/java/com/socialmediaraiser/twitterbot/GoogleAuthorizeUtil.java]: Yes it was a mistake (check the next line haha)

**Pull request merged**